### DISCUSSION TITLE
how to maintain or have multiple app specific passwords in Jenkins or can I rename env variable name?

### DISCUSSION THREAD

**Discussion by praveens96** [fastlane/fastlane#18640] [General]
Hello,
we have CI Mac slave where the .ipa's would be generated. we are using it for mobile app till now. we are doing a tvOS app now and want to use same slave for building its ipa as well. a pre-requisite for Jenkins is to have the environmental variable FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD. 
but how to have two such variables? or can I mention in match file/fast file that to check for different env variable name?
any suggestions would be appreciated.

**Comment by max-ott** [comment]
You'd have to set the env var before running the lane. I don't think you can use custom env vars.
